xml_inscriptions/transcribed/NY.NY.MMA.L.18.145.1aAndb.xml

```diff
@@ -0,0 +1,192 @@
+                        </provenance>
+
+
+                    </history>
+                </msDesc>
+                <!--  *************************************  <listBibl>  *********************************  -->
+                <listBibl>
+                    <bibl>
+                        <ptr type="rest-of-bibl" target="#Goodenough"/>
+                        <biblScope unit="page">111-112</biblScope>
+                    </bibl>
+                    <bibl>
+                        <ptr type="rest-of-bibl" target="#JIWE"/>
+                        <biblScope unit="item">2.591</biblScope>
+                    </bibl>
+                    <bibl>
+                        <ref type="external" target="https://www.metmuseum.org/art/collection/search/465921">Metropolitan Museum Web Entry</ref>
+                    </bibl>
+                </listBibl>
+            </sourceDesc>
+        </fileDesc>
+
+        <!--  *************************************  <encodingDesc>  ********************************* -->
+
+        <encodingDesc>
+
+                <xi:include
+                    href="http://library.brown.edu/usep_data/resources/include_taxonomies.xml">
+                    <xi:fallback>
+                        <p>Taxonomies for US Epigraphy controlled values</p>
+                    </xi:fallback>
+                </xi:include>
+
+        </encodingDesc>
+
+        <profileDesc> <!-- not used --> </profileDesc>
+
+
+        <!--  *************************************  <revisionDesc>  ********************************* -->
+    	<!-- Record all changes here -->
+
+        <revisionDesc>
+            <change when="2021-03-02" who="Itamar Levin">Transcription, translation, and
+                metadata</change>
+        </revisionDesc>
+
+    </teiHeader>
+
+    <!--  *************************************  <facsimile>  *********************************  -->
+        <!-- Used to record images of an inscription.   -->
+
+    <facsimile>
+        <surface>
+            <desc/>
+            <graphic url="NY.NY.MMA.L.18.145.1aAndb.jpg"/>
+        </surface>
+    </facsimile>
+
+    <text>
+        <body>
+            <div type="edition">
+            <ab>
+                <lb n="1"/><gap reason="omitted" extent="unknown"></gap>ci <choice><sic>bibas</sic><corr>vivas</corr></choice> cum eulogia conp<supplied reason="lost" cert="low">are</supplied><gap reason="lost" extent="unknown"></gap>
+            </ab></div>
+            
+            <div type="translation">
+            <p>May you live with equal blessing.</p></div>
+
+            <div type="commentary">
+            <p>This bowl-fragment is a rare example of the gold-glasses found in the catacombs where
+                    they served as grave-markers. Gold glasses were common in 3rd-4th century Rome,
+                    where this bowl's elaborate technique supports dating it to the 4th century.
+                    The meaning of the inscription is dubious; "bibas" (may you drink) or an
```

Comment:
The phonetic shift is typical of Late antiquity more generally, not just in Jewish inscriptions (we see this in Greek especially, where β has already started moving towards the modern Greek phoneme, and it spreads into Latin shortly after).